src/quo2/components/community/discover_card.cljs

```diff
@@ -0,0 +1,134 @@
+            [quo2.foundations.colors :as colors]
+            [status-im.ui.screens.communities.styles :as styles]
+            [status-im.ui.components.react :as react]))
+
+;; Mirrors placeholders images.
+;; TO BE replaced when real data is available
+(def images
+  {:joined     [{:id 1 :row-images    [{:id 1   :image   ""}
+                                       {:id 2   :image   ""}
+                                       {:id 3   :image   ""}
+                                       {:id 4   :image   ""}]}
+                {:id 2 :row-images    [{:id 1   :image   ""}
+                                       {:id 2   :image   ""}
+                                       {:id 3   :image   ""}]}]
+   :not-joined [{:id 1 :column-images [{:id 1   :image   ""}]}
+                {:id 2 :column-images [{}]}]})
+
+(defn card-title-and-description [title description joined]
+  [react/view
+   {:flex               1
+    :padding-top        (if joined 8 10)
+    :padding-bottom     (if joined 8 2)
+    :border-radius      12}
+   [react/view {:flex                1
+                :padding-horizontal  12}
+    [text/text {:accessibility-label :community-name-text
+                :ellipsize-mode      :tail
+                :number-of-lines     1
+                :weight              :semi-bold
+                :size                (if joined
+                                       :paragraph-1
+                                       :heading-2)}
+     title]
+    [text/text {:accessibility-label :community-name-text
+                :ellipsize-mode      :tail
+                :number-of-lines     1
+                :color               (colors/theme-colors
+                                      colors/neutral-50
+                                      colors/neutral-40)
+                :weight               :regular
+                :size                 (if joined
+                                        :paragraph-2
+                                        :paragraph-1)}
+     description]]])
+
+(defn placeholder-list-images [{:keys [images width height border-radius joined]
+                                :or {joined false}}]
+  [react/view
+   [react/view (if joined
+                 {:justify-content    :center}
+                 {:flex-direction     :row})
+    (for [{:keys [id]} images]
+      ^{:key id}
+      [react/view {:border-radius border-radius
+                   :margin-top    (if joined 4 8)
+                   :margin-right  (if joined
+                                    4 8)
+                   :width         width
+                   :height        height
+                   :background-color colors/neutral-10}])]])
+
+(defn placeholder-row-images [{:keys [first-image last-image images width height
+                                      border-radius joined]}]
+  [react/view (when-not joined
+                {:flex-direction :row})
+   (when first-image
+     [react/view {:border-top-right-radius    (when-not joined
```

Comment:
how come first-image is being used here? 🤔

Reply:
@flexsurfer suggested since this is a placeholder its okay to just use `rn/view` instead of `rn/image`. Using the `image` component would lead havings lots of unnecessary images in the resources directory